application/models/User.php

```diff
@@ -540,13 +540,8 @@ public function getManagementButtons()
             'linkAttributes'   => [
                 'data-href' => $editUrl,
             ],
-            'enabledCondition' =>
-                ($permission_superadmin_read
```

Comment:
I guess this situations is now talen care on the `userManager->canEdit()` method, right?

Reply:
yes

Reply:
> I guess this situations is now talen care on the `user->canEdit()` method, right?

It's ALL the point : to move to the same function for testing and show …